doc/contributing.rst

```diff
@@ -33,8 +33,8 @@ code in order to contribute, you can follow those steps:
 
      $ git clone git@github.com:mne-tools/mne-python.git
      $ cd mne-python
-     $ conda env create -f dev_environment.yml
-     $ conda activate mne-dev-py36
+     $ conda env create -f environment.yml
+     $ conda activate mne
      $ pip install -e .
```

Comment:
you are missing all the dev dependencies that need to be installed by hand

Reply:
@massich which one do you have in mind? our environment.yml installs pytest etc.

Reply:
https://github.com/mne-tools/mne-python/pull/5768/files#r242466086